frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/services/MfaResetIPVAuthorizationServiceTest.java

```diff
@@ -0,0 +1,239 @@
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.jwk.JWK;
+import com.nimbusds.jose.jwk.RSAKey;
+import com.nimbusds.jose.util.Base64URL;
+import com.nimbusds.jwt.EncryptedJWT;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
+import com.nimbusds.oauth2.sdk.ResponseType;
+import com.nimbusds.oauth2.sdk.id.State;
+import com.nimbusds.oauth2.sdk.id.Subject;
+import com.nimbusds.oauth2.sdk.token.BearerAccessToken;
+import com.nimbusds.openid.connect.sdk.OIDCClaimsRequest;
+import com.nimbusds.openid.connect.sdk.claims.ClaimsSetRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedConstruction;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import uk.gov.di.audit.AuditContext;
+import uk.gov.di.authentication.shared.entity.ResponseHeaders;
+import uk.gov.di.authentication.shared.entity.Session;
+import uk.gov.di.authentication.shared.helpers.IdGenerator;
+import uk.gov.di.authentication.shared.helpers.NowHelper;
+import uk.gov.di.authentication.shared.serialization.Json;
+import uk.gov.di.authentication.shared.services.AuditService;
+import uk.gov.di.authentication.shared.services.CloudwatchMetricsService;
+import uk.gov.di.authentication.shared.services.ConfigurationService;
+import uk.gov.di.authentication.shared.services.RedisConnectionService;
+import uk.gov.di.authentication.shared.services.SerializationService;
+import uk.gov.di.authentication.shared.services.TokenService;
+import uk.gov.di.authentication.sharedtest.helper.TestClockHelper;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.RSAPublicKey;
+import java.text.ParseException;
+import java.time.temporal.ChronoUnit;
+import java.util.Base64;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.di.authentication.frontendapi.domain.FrontendAuditableEvent.REVERIFY_AUTHORISATION_REQUESTED;
+
+class MfaResetIPVAuthorizationServiceTest {
+    private static final JWSAlgorithm TEST_SIGNING_ALGORITHM = JWSAlgorithm.ES256;
+    private static final String TEST_MFA_RESET_SCOPE = "reverification";
+    private static final String TEST_STATE_STORAGE_PREFIX = "state:";
+    private static final String TEST_STATE_VALUE = "testState";
+    private static final String TEST_SESSION_ID = "testSessionId";
+    private static final Session TEST_SESSION = new Session(TEST_SESSION_ID);
+    private static final String TEST_CLIENT_SESSION_ID = "journeyId";
+    private static final String TEST_INTERNAL_SUBJECT_IDENTIFIER = "urn:some:identifier";
```

Comment:
This is the place I meant about reusing the common variable/keeping them similar values

Reply:
Have updated!